dist/functional-test.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+
+ETCD_IMG="quay.io/coreos/etcd:v3.0.3"
```

Comment:
Is it better to us a fixed version here or to just take :latest? Tracking latest might give us an early heads-up if there's a problem using flannel with the latest etcd release?